compiler/rustc_passes/src/check_attr.rs

```diff
@@ -686,7 +688,8 @@ impl<'tcx> CheckAttrVisitor<'tcx> {
                         Attribute::Parsed(
                             AttributeKind::Deprecation { .. }
                             | AttributeKind::Repr { .. }
-                            | AttributeKind::Align { .. },
+                            | AttributeKind::Align { .. }
+                            | AttributeKind::MustUse { .. },
```

Comment:
I've added `MustUse` here but not removed it from the `ALLOW_LIST` above, because you also seem to not have done this for the other attributes. Is this correct?

Reply:
I've just rewritten the naked attribute, so this shouldn't be necessary anymore. but dw, I'll do the rebase on there